### PR TITLE
Partial TL translations for client-p4.asc - 1,493 words

### PR DESCRIPTION
**Pull request by josteinfrancois**
I have translated 1,493 words (code excluded) for this pull request. Please help me review my work. Thank you.

### CODE REVIEW

File: book/09-git-and-other-scms/sections/client-p4.asc [hunk 1]
```diff
@@ -17,21 +17,21 @@ Ang pangalawa ay ang git-p4, isang client-side na tulay na hahayaan kang gumamit
 (((Perforce, Pagsasanib ng Git)))
 Ang Perforce ay nagbibigay ng isang produkto na tinatawag na Pagsasanib ng Git (makukuha sa http://www.perforce.com/git-fusion[]), na pinagsasabay ang isang Perforce na server sa Git na mga repositoryo sa panig ng server.
 
-====== Setting Up
+====== Pag-set Up
 
-For our examples, we'll be using the easiest installation method for Git Fusion, which is downloading a virtual machine that runs the Perforce daemon and Git Fusion.
-You can get the virtual machine image from http://www.perforce.com/downloads/Perforce/20-User[], and once it's finished downloading, import it into your favorite virtualization software (we'll use VirtualBox).
+Para sa ating mga halimbawa, gagamit tayo ng pinakamadaling pag-install na pamamaraan para sa Git Fusion, na nagda-donwload ng isang birtwal na makina na nagpapatakbo ng Perforce daemon at Git Fusion.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-p4.asc]: no need to change virtual. "birtwal" change back to virtual.

File: book/09-git-and-other-scms/sections/client-p4.asc [hunk 2]
```diff
@@ -17,21 +17,21 @@ Ang pangalawa ay ang git-p4, isang client-side na tulay na hahayaan kang gumamit
...
+Maaari mong kunin ang imahe ng birtwal na makina mula sa http://www.perforce.com/downloads/Perforce/20-User[], at kapag ito ay natapos nang mag-download, i-import ito sa iyong paboritong virtualization na software (gagamit tayo ng VirtualBox).
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-p4.asc]: birtwal to virtual

File: book/09-git-and-other-scms/sections/client-p4.asc [hunk 3]
```diff
@@ -17,21 +17,21 @@ Ang pangalawa ay ang git-p4, isang client-side na tulay na hahayaan kang gumamit
...
 
-Upon first starting the machine, it asks you to customize the password for three Linux users (`root`, `perforce`, and `git`), and provide an instance name, which can be used to distinguish this installation from others on the same network.
-When that has all completed, you'll see this:
+Sa unang pagsisimula sa makina, tatanungin ka nito na i-customize ang password para sa tatlong Linux na mga user (`root`, `perforce`, at `git`), at magbigay ng isang instance na pangalan, na maaaring gamitin upang makilala ang pag-install na ito mula sa mga iba sa parehong network.
+Kapag nakumpleto ng iyon ang lahat, makikita mo ito:
 
-.The Git Fusion virtual machine boot screen.
-image::images/git-fusion-boot.png[The Git Fusion virtual machine boot screen.]
+.Ang boot screen ng birtwal na makina ng Git Fusion
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-p4.asc]: birtwal to virtual

File: book/09-git-and-other-scms/sections/client-p4.asc [hunk 4]
```diff
@@ -98,9 +98,9 @@ $ tree
 498 directories, 287 files
 ----
 
-The `objects` directory is used internally by Git Fusion to map Perforce objects to Git and vice versa, you won't have to mess with anything in there.
-There's a global `p4gf_config` file in this directory, as well as one for each repository – these are the configuration files that determine how Git Fusion behaves.
-Let's take a look at the file in the root:
+Ang `objects` na direktoryo ay panloob na ginagamit ng Git Fusion upang i-map ang mga object ng Perforce sa Git at kabaligtaran, hindi mo na kailangang guluhin ang anumang nandoon.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-p4.asc]: i-map to i-mapa

Comment (mind-blast) [reply book/09-git-and-other-scms/sections/client-p4.asc]: object  to bagay

File: book/09-git-and-other-scms/sections/client-p4.asc [hunk 5]
```diff
@@ -144,11 +144,11 @@ git-branch-name = master
...
-This is a mapping between a Perforce branch and a Git branch.
-The section can be named whatever you like, so long as the name is unique.
-`git-branch-name` lets you convert a depot path that would be cumbersome under Git to a more friendly name.
-The `view` setting controls how Perforce files are mapped into the Git repository, using the standard view mapping syntax.
-More than one mapping can be specified, like in this example:
+Ito ay isang pag-map sa pagitan ng isang Perfoce na branch at isang Git na branch.
+Ang seksyon ay maaaring pangalanan sa kung anuman ang gugustuhin mo, hangga't ang pangalan ay natatangi.
+Ang `git-branch-name` ay hahayaan kang palitan ang isang depot na landas na maaaring maging masalimuot sa ilalim ng Git ng isang mas palakaibigan na pangalan.
+Ang `view` na setting ay namamahala kung paano na-map ang mga file ng Perforce sa Git na repositoryo, gamit ang syntax ng standard view ng pag-map.
+Higit pa sa isang pag-map ang maaaring matukoy, kagaya ng halimbawang ito:
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-p4.asc]: pag-map to pagmapa

File: book/09-git-and-other-scms/sections/client-p4.asc [hunk 6]
```diff
@@ -158,12 +158,12 @@ view = //depot/project1/main/... project1/...
        //depot/project2/mainline/... project2/...
 ----
 
-This way, if your normal workspace mapping includes changes in the structure of the directories, you can replicate that with a Git repository.
+Sa paraang ito, kung ang iyong normal na pag-map ng workspace ay nagsasama ng mga pagbabago sa istraktura ng mga direktoryo, maaari mong kopyahin iyon gamit ang isang Git na repositoryo.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-p4.asc]: pag-map to pagmapa

File: book/09-git-and-other-scms/sections/client-p4.asc [hunk 7]
```diff
@@ -158,12 +158,12 @@ view = //depot/project1/main/... project1/...
...
 
-The last file we'll discuss is `users/p4gf_usermap`, which maps Perforce users to Git users, and which you may not even need.
-When converting from a Perforce changeset to a Git commit, Git Fusion's default behavior is to look up the Perforce user, and use the email address and full name stored there for the author/committer field in Git.
-When converting the other way, the default is to look up the Perforce user with the email address stored in the Git commit's author field, and submit the changeset as that user (with permissions applying).
-In most cases, this behavior will do just fine, but consider the following mapping file:
+Ang huling file na tatalakayin natin ay ang `users/p4gf_usermap`, na nagma-map ng Perforce na mga user, at maaaring hindi mo na kakailanganin.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-p4.asc]: na nagma-map to kung saan nagmamapa

Comment (mind-blast) [reply book/09-git-and-other-scms/sections/client-p4.asc]: user to gumagamit

File: book/09-git-and-other-scms/sections/client-p4.asc [hunk 8]
```diff
@@ -158,12 +158,12 @@ view = //depot/project1/main/... project1/...
...
+Kapag ikaw ay ng nagpapalit mula sa isang Perforce na changeset patungo sa isang Git na commit, ang default na pag-uugali ng Git Fusion ay ang paghahanap sa Perforce na user, at gagamit ng email address at buong pangalan na naka-imbak doon para sa may-akda/committer na field sa Git.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-p4.asc]: committer to taga-commit

Comment (mind-blast) [reply book/09-git-and-other-scms/sections/client-p4.asc]: user to gumagamit

Comment (mind-blast) [reply book/09-git-and-other-scms/sections/client-p4.asc]: field  to patlang

File: book/09-git-and-other-scms/sections/client-p4.asc [hunk 9]
```diff
@@ -173,20 +173,20 @@ bob employeeX@example.com "Anon X. Mouse"
 joe employeeY@example.com "Anon Y. Mouse"
 ----
 
-Each line is of the format `<user> <email> "<full name>"`, and creates a single user mapping.
-The first two lines map two distinct email addresses to the same Perforce user account.
-This is useful if you've created Git commits under several different email addresses (or change email addresses), but want them to be mapped to the same Perforce user.
-When creating a Git commit from a Perforce changeset, the first line matching the Perforce user is used for Git authorship information.
+Ang bawat linya ay naka-format na `<user> <email> "<full name>"`, at lumilikha ng isang solong user na pag-map.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-p4.asc]: pag-map to pagmamapa

File: book/09-git-and-other-scms/sections/client-p4.asc [hunk 10]
```diff
@@ -173,20 +173,20 @@ bob employeeX@example.com "Anon X. Mouse"
...
+Ang unang dalawang mga linya ay nagma-map ng dalawang naiibang mga email address sa parehong Perforce user na account.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-p4.asc]: nagma-map to nagmapa

**Comment by mind-blast** [comment]
Suggested some words. Hope you can check them out.

**Comment by josteinfrancois** [comment]
Hello @robin-ho . I finished updating my work based on your suggestions. Thank you.

**Pull request merged**